src/_tutorials/server/get-started.md

```diff
@@ -1,263 +1,237 @@
-you can create a basic app using standard Dart templates.
+This creates a minimal Dart app that has the following:
 
-If you're using a [Dart-savvy IDE][ide], follow these instructions:
+* One main Dart source file: `bin/main.dart`. This contains one top-level
+  `main()` function. This is the entry point for your app.
+* One additional Dart file: `lib/cli.dart`. This contains the functionality of
+  the app, and is imported by the `main.dart` file.
+* One `pubspec.yaml` file. This contains the app's metadata, including
+  information about which [packages](/tools/pub/get-started) the app depends on
+  and which versions of those packages are required.
 
-1. Create a new Dart project called `hello_world`,
-   using files from the template named
-   **console-full**, which has the description 
-   **A command-line application sample.**
+## 5. Run the app
 
-1. If the IDE doesn't automatically download the packages
-   that the app depends on, make the IDE run `pub get`.
+First we need to get the dependencies of the app using the
+[`pub`](/tools/pub/cmd) command:
 
-If you're using the command line, follow these instructions:
+```terminal
+> pub get
+```
 
-1. Get [Stagehand.]({{site.pub}}/packages/stagehand)
+To run the app from the command line, use the Dart VM by running the
+[`dart`][/tools/dart-vm] command:
 
-1. Use Stagehand to get the files, and then
-   use the pub tool to download dependent packages.
-   The commands you use should be something like this:
-   
-   ```terminal
-   mkdir hello_world
-   cd hello_world
-   stagehand console-full
-   pub get
-   ```
+```terminal
+> dart bin/main.dart
+Hello world: 42!
+```
 
-Some of the files and directories in the hello_world app
-include the following:
+## 6. Add custom code to the app
 
-`analysis_options.yaml`
-: Defines lint rules enforced for this project.
-  By default, this file specifies the set of rules used for Google projects,
-  but you can [customize these options](/guides/language/analysis-options).
+Let's customize the app you just created:
 
-`bin`
-: Contains the source files for the app.
-  The main Dart file for this example is `bin/main.dart`.
+ 1. Edit `lib/cli.dart` to return a different result:
 
-`lib`
-: Contains library code.
-  In this directory, `hello_world.dart` defines
-  a simple `calculate()` method.
+    ```dart
+    int calculate() {
+      return -1;
+    }
+    ```
+ 1. Save your changes.
 
-`pubspec.lock`
-: A generated file that specifies the version numbers
-  of the packages on which the app depends.
+ 1. Re-run the main entry-point of your app:
 
-`pubspec.yaml`
-: Declares which packages your app needs.
+    ```terminal
+    > dart bin/main.dart
+    Hello world: -1
+    ```
 
-`test`
-: Contains test code for the hello_world app.
+More information:
+[Write Command-Line Apps](/tutorials/server/cmdline)
 
-`.dart_tool`
-: Support files used by `pub` and other Dart tools. You can ignore this.
+## 7. Use DevTools to debug the app
```

Comment:
I think we should (1) put the relevant, foolproof instructions here (like we do for stagehand) and (2) improve the instructions. There's no help at that site for running the `dart` command, so I wasn't sure what to do.

Delete this step for now?

Reply:
Deleting, and filed https://github.com/dart-lang/site-www/issues/1522 to track re-adding